examples/jaxsim_walking.ipynb

```diff
@@ -0,0 +1,594 @@
+    "# Flag to disable JAX JIT compilation\n",
+    "# os.environ[\"JAX_DISABLE_JIT\"] = \"True\"\n",
+    "# Flag to solve MUMPS hanging\n",
+    "os.environ[\"OMP_NUM_THREADS\"] = \"1\"\n",
+    "\n",
+    "from comodo.jaxsimSimulator import JaxsimSimulator, JaxsimContactModelEnum\n",
+    "from comodo.robotModel.robotModel import RobotModel\n",
+    "from comodo.robotModel.createUrdf import createUrdf\n",
+    "from comodo.centroidalMPC.centroidalMPC import CentroidalMPC\n",
+    "from comodo.centroidalMPC.mpcParameterTuning import MPCParameterTuning\n",
+    "from comodo.TSIDController.TSIDParameterTuning import TSIDParameterTuning\n",
+    "from comodo.TSIDController.TSIDController import TSIDController"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ==== Load the stickbot model ====\n",
+    "\n",
+    "# Getting stickbot urdf file and convert it to string\n",
+    "urdf_robot_file = tempfile.NamedTemporaryFile(mode=\"w+\")\n",
+    "url = \"https://raw.githubusercontent.com/icub-tech-iit/ergocub-gazebo-simulations/master/models/stickBot/model.urdf\"\n",
+    "urllib.request.urlretrieve(url, urdf_robot_file.name)\n",
+    "# Load the URDF file\n",
+    "tree = ET.parse(urdf_robot_file.name)\n",
+    "root = tree.getroot()\n",
+    "\n",
+    "# Convert the XML tree to a string\n",
+    "robot_urdf_string_original = ET.tostring(root)\n",
+    "\n",
+    "create_urdf_instance = createUrdf(\n",
+    "    original_urdf_path=urdf_robot_file.name, save_gazebo_plugin=False\n",
+    ")\n",
+    "\n",
+    "joint_names = [\n",
+    "    \"r_shoulder_pitch\",  # 0\n",
+    "    \"r_shoulder_roll\",  # 1\n",
+    "    \"r_shoulder_yaw\",  # 2\n",
+    "    \"r_elbow\",  # 3\n",
+    "    \"l_shoulder_pitch\",  # 4\n",
+    "    \"l_shoulder_roll\",  # 5\n",
+    "    \"l_shoulder_yaw\",  # 6\n",
+    "    \"l_elbow\",  # 7\n",
+    "    \"r_hip_pitch\",  # 8\n",
+    "    \"r_hip_roll\",  # 9\n",
+    "    \"r_hip_yaw\",  # 10\n",
+    "    \"r_knee\",  # 11\n",
+    "    \"r_ankle_pitch\",  # 12\n",
+    "    \"r_ankle_roll\",  # 13\n",
+    "    \"l_hip_pitch\",  # 14\n",
+    "    \"l_hip_roll\",  # 15\n",
+    "    \"l_hip_yaw\",  # 16\n",
+    "    \"l_knee\",  # 17\n",
+    "    \"l_ankle_pitch\",  # 18\n",
+    "    \"l_ankle_roll\",  # 19\n",
+    "]\n",
+    "\n",
+    "urdf_robot_string = create_urdf_instance.write_urdf_to_file()\n",
+    "robot_model_init = RobotModel(urdf_robot_string, \"stickBot\", joint_names)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ==== Set simulation parameters ====\n",
+    "\n",
+    "T = 6.0\n",
+    "js_dt = 0.000_5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ==== Compute initial configuration ====\n",
+    "\n",
```

Comment:
This fixes `AttributeError: 'RobotModel' object has no attribute 'corner_0'`
```suggestion
    "\n",
    "robot_model_init.set_foot_corner(np.asarray([0.1, 0.05, 0.0]),np.asarray([0.1, -0.05, 0.0]),np.asarray([-0.1, -0.05, 0.0]),np.asarray([-0.1, 0.05, 0.0]))\n",
```

Reply:
I confirm that this fixes the problem of https://github.com/ami-iit/comodo/pull/20#pullrequestreview-2365661553. While reviewing this PR, I missed this comment. Apparently in the `Files changed` tab, diffs on ipynb are not reported when the file view is the rich one.

Reply:
Done in fcea6d97f9e056b120d26757a23051be49dac5c1

Reply:
> I confirm that this fixes the problem of [#20 (review)](https://github.com/ami-iit/comodo/pull/20#pullrequestreview-2365661553). While reviewing this PR, I missed this comment. Apparently in the `Files changed` tab, diffs on ipynb are not reported when the file view is the rich one.

Yes it is a bit cumbersome to locate comments in notebooks